pants-plugins/internal_plugins/releases/register.py

```diff
@@ -40,7 +40,10 @@ def _branch_name(cls, version: Version) -> str:
         suffix = version.public[len(version.base_version) :]
         components = version.base_version.split(".") + [suffix]
         if suffix != "" and not (
-            suffix.startswith("rc") or suffix.startswith("a") or suffix.startswith(".dev")
+            suffix.startswith("rc")
+            or suffix.startswith("a")
+            or suffix.startswith("b")
+            or suffix.startswith(".dev")
```

Comment:
This is just on the boundary where I might consider a regular expression, or possibly:
```python
any(suffix.startswith(s) for s in ["rc", "a", "b", ".dev"])
```

Reply:
You can pass it a tuple instead :)

```
>>> "foo".startswith(("f", "a", "c"))
True
```